frontend/README.md

```diff
@@ -0,0 +1,68 @@
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
```

Comment:
this readme still valid?

Reply:
🤦, Embarrassing mistake, but resolved now.